test/integration/ssr-ctx/pages/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
-
-const Idk = React.createContext(null)
+import { useIdk } from '../context'
 
 const Page = () => {
+  const idk = useIdk()
+
+  console.log(idk)
```

Comment:
You left a console.log here 😉

Reply:
This was on purpose, it's to see when the test is failing while running